app/models/collection.rb

```diff
@@ -2,6 +2,10 @@
 class Collection < ActiveFedora::Base
   include ::Hyrax::CollectionBehavior
 
+  property :ead, predicate: ::Tufts::Vocab::Tufts.has_description do |index|
```

Comment:
Does this predicate need to be entered in `app/lib/tufts/vocab/tufts.rb`?  I'm not sure what the purpose of registering terms in that file is.

It seems like I can have any arbitrary term without adding it to that file, so maybe it's not necessary:
```ruby
2.3.4 :002 > ::Tufts::Vocab::Tufts.blergg
 => #<RDF::Vocabulary::Term:0x3fcf3bce12d4 URI:http://dl.tufts.edu/terms#blergg>
```

@no-reply What is your opinion?

Reply:
@mark-dce and I looked at it and we think I should add it to that file. Good catch, @val99erie .

Reply:
@val99erie - yes, I think you're correct, we can create a new predicate name anywhere we want in the codebase.  For clarity and in case we want to refer to the predicate elsewhere, I think it's probably good practice to "register" it in `app/lib/tufts/vocab/tufts.rb` which is sort of the canonical place for us to say "Here's the things we've defined as being part of the Tufts local RDF vocabulary"